### DISCUSSION TITLE
How to use `<Title />` or `<TitleRenderer />`?

### DISCUSSION THREAD

**Discussion by QuentinRoy** [slidevjs/slidev#1433] [Q&A] [answered]
I am rather confused at the "`titles`" built in components. I couldn't find a way to make them work. I have some advanced markup to dispay my title so I am not using `# title` for most of my slides. As a consequences the slides are all considered untitled. How can I avoid to enter the title both in each slide's frontmatter and its content? I thought I could add the title in the frontmatter then use `<Title />` or `<TitleRenderer />` to insert it in my slide (as below), but clearly it doesn't work. I don't quite understand [the documentation](https://sli.dev/builtin/components#titles) and I couldn't find my way in the source code. Did anyone figured out how to do this?

```mdx
---
title: My slide
---
<h1><Title /> -- <span>Something</span></h1>
```

**Comment by QuentinRoy** [comment, marked as answer]
OK, it took me some time but I finally got it. #1469 should fix my use case.

**Comment by mtrocadomoreira** [comment]
Thank you, I was going crazy looking for a solution to a similar problem 😭
